scripts/fork-verify.sh

```
@@ -0,0 +1,417 @@
+# fork-verify.sh - Verifica integridad de fork de agente
+# Uso: ./scripts/fork-verify.sh <target_dir> [--strict]
+# Ejemplo: ./scripts/fork-verify.sh .claude
+#          ./scripts/fork-verify.sh .claude --strict
+
+set -uo pipefail
+
+# Configuración
+TARGET_DIR="${1:-.}"
+STRICT="${2:-}"
+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+PROJECT_ROOT="$(cd "$SCRIPT_DIR/.." && pwd)"
```

Shellcheck (0.11.0)

[warning] PROJECT_ROOT appears unused. Verify use (or export if used externally).

(SC2034)